dataland-frontend/tests/e2e/utils/FakeFixtureUtils.ts

```diff
@@ -151,6 +151,14 @@ export class Generator {
     return faker.company.buzzNoun();
   }
 
+  randomParagraph(): string | null {
+    return this.valueOrNull(this.guaranteedParagraph());
+  }
+
+  guaranteedParagraph(): string {
+    return faker.lorem.paragraphs({ min: 1, max: 5 });
```

Comment:
`faker.lorem.paragraphs` actually generates multiple paragagraphs. `faker.lorem.paragraph` however generates a paragraph with the provided number of sentences. Theres is also `faker.lorem.lines` which generates multiple lines. I think one of the latter is rather what you want here.

Reply:
Ok I looked at all three functions (`.paragraphs`, `.paragraph` and also `.lines`) and what they do.
Now I think `.paragraphs` is indeed the best thing to use here.

Reason:

We want to generate very long texts that also have line breaks.
`.lines()`does line breaks, but it only writes one short sentence per line. We want more text than that.
`.paragraph()` has multiple sentences, but it does not do line breaks.

However, `.paragraphs()` does line breaks AND it also writes a lot of sentences per line.
So it is exactly what we want here.


Still I would suggest to rename the function to `guranteedParagraphs` with a "s" at the end.

Is that ok with you?

Reply:
Agreed